modules/c3-assisted-installer-overview.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-overview_{context}"]
+= Overview
+
+You can install {product-title} on {oci-c3-no-rt} by using the {ai-full}. 
+
+If you want to set up the cluster manually or using other automation tools, or if you are working in a disconnected environment, you can use the Red Hat Agent-based Installer for the installation. For details, see 'Installing a cluster on {oci-c3} by using the Agent-based Installer'. *[Will replace with link when I have it. Copied the paragraph from OCI, is the first part of the first sentence still correct?]*
+
+.Installation process
+
+The installation process involves building two hosts within the designated compartment of the {product-title} cluster:
+
+* The first instance is a `bastion` host, which is used to run two Terraform scripts:
++
+--
+** The first script builds IAM Resources in the {oci} Home region of the {oci-c3-short} system (two Dynamic Groups and one Policy).
+
+** The second script builds the infrastructure resources on the {oci-c3-short} system to support the RHOCP cluster (the RHOCP VCN,  public and private subnets, load balancers, Internet GW, NAT GW, and DNS server). It includes all the resources needed to activate the control plane nodes and compute nodes that form a cluster. 
+--
++
+The bastion host is installed in the designated RHOCP Compartment and configured to communicate through a designated {oci-c3-short} DRG Subnet or Internet GW Subnet within the {oci-c3-short} parent tenancy. 
+
+* The second instance is the `jumpserver`. The jumpserver is configured as a web server to communicate to the RHOCP cluster. The jumpserver is installed with the {product-title} client CLI, within the {product-title} VCN built by the Terraform scripts.
+
+The installation process subsequently provisions three control plane (master) nodes and three compute (worker) noodes, together with the external and internal Load Balancers that form the cluster. 
+
+.Main steps
+
+The main steps of the procedure are as follows:
```

Comment:
Is the whole process day 1 or day 2? I haven't related to day 2 so maybe I need to add an explanation.